<ui version="4.0" >
 <class>SingleFileResourceConfigDialog</class>
 <widget class="QWidget" name="SingleFileResourceConfigDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>386</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="KTabWidget" name="ktabwidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>File</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>File Name</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>&amp;Filename:</string>
              </property>
              <property name="buddy" >
               <cstring>kcfg_Path</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KUrlRequester" name="kcfg_Path" />
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="statusLabel" >
            <property name="text" >
             <string>Status:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Select the file which content should be represented by this resource. If the file does not exist yet it will be created. You can also specify an URL to a remote file, however monitoring the file for changes will not work in this case.</string>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Access Rights</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <widget class="QCheckBox" name="kcfg_ReadOnly" >
            <property name="text" >
             <string>Read Only</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>If you enable the read-only mode no changes will be written to the file selected above. Read-only mode will be automatically enabled if you do not have write access to the file or the file is on a remote server that does not support write access.</string>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>4</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Tuning</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5" >
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>The options on this page allow you to change parameters that balance data safety and consistency against performance. In general you should be careful with changing anything here, the defaults are good enough in most cases.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <item>
          <widget class="QLabel" name="autosaveLabel" >
           <property name="text" >
            <string>Autosave delay:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KIntNumInput" name="kcfg_AutosaveInterval" >
           <property name="value" >
            <number>1</number>
           </property>
           <property name="minimum" >
            <number>0</number>
           </property>
           <property name="suffix" >
            <string>min</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_MonitorFile" >
         <property name="text" >
          <string>Enable file &amp;monitoring</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>138</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_ReadOnly</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_AutosaveInterval</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>273</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_ReadOnly</sender>
   <signal>toggled(bool)</signal>
   <receiver>autosaveLabel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>273</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel" >
     <x>56</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
